src/routes/console/project-[project]/auth/security/updatePasswordHistory.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form, InputNumber } from '$lib/elements/forms';
+    import FormList from '$lib/elements/forms/formList.svelte';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdkForConsole } from '$lib/stores/sdk';
+    import { project } from '../../store';
+
+    const projectId = $project.$id;
+    let passwordHistory = $project.authPasswordHistory ?? 0;
+
+    async function updatePasswordHistoryLimit() {
```

Comment:
This one as well

Reply:
I believe it's because our version of Appwrite is not the latest